server/routes/barcode/contacts/CreateContactByPrisonerNameController.ts

```diff
@@ -78,4 +84,20 @@ export default class CreateContactByPrisonerNameController {
 
     req.session.recipients.push(recipient)
   }
+
+  parsePrisonerDob(req: Request): Date | undefined {
+    const dobDay: string = req.body['prisonerDob-day'] ? (req.body['prisonerDob-day'] as string).padStart(2, '0') : ''
+    const dobMonth: string = req.body['prisonerDob-month']
+      ? (req.body['prisonerDob-month'] as string).padStart(2, '0')
+      : ''
+    const dobYear: string = req.body['prisonerDob-year'] ? (req.body['prisonerDob-year'] as string) : ''
+    if (dobDay === '' && dobMonth === '' && dobYear === '') {
+      return undefined
+    }
+    try {
+      return moment(`${dobDay}-${dobMonth}-${dobYear}`, 'DD-MM-YYYY', true).toDate()
+    } catch (error) {
+      return undefined
+    }
```

Comment:
Ah I need to remoe this try catch - moment handles the error and creates a Data(NaN)